src/pages/ProfilesExplorerView/components/SceneExploreServiceFlameGraph/components/SceneFunctionDetailsPanel/domain/__tests__/convertPprofToFunctionDetails.spec.ts

```diff
@@ -384,7 +420,7 @@ describe('convertPprofToFunctionDetails(fnName, profile)', () => {
               [134, { cum: 1650000000, flat: 1650000000, line: 134 }],
               [120, { cum: 560000000, flat: 0, line: 120 }],
               [137, { cum: 820000000, flat: 820000000, line: 137 }],
-              [142, { cum: 400000000, flat: 10000000, line: 142 }],
```

Comment:
This is expected, as the sample contains inlining hence the spec was slightly incorrect.